Ref/Top/instances.fpp

```diff
@@ -226,122 +111,29 @@ module Ref {
   @ Note: Here we have TCP reliable uplink and UDP (low latency) downlink
   instance comm: Drv.ByteStreamDriverModel base id 0x4000 \
     type "Drv::TcpClient" \
-    at "../../Drv/TcpClient/TcpClient.hpp" \
-  {
-
-    phase Fpp.ToCpp.Phases.configConstants """
-    enum {
-      PRIORITY = 100,
-      STACK_SIZE = Default::stackSize
-    };
-    """
-
-    phase Fpp.ToCpp.Phases.startTasks """
-    // Initialize socket server if and only if there is a valid specification
-    if (state.hostName != nullptr && state.portNumber != 0) {
-        Os::TaskString name("ReceiveTask");
-        // Uplink is configured for receive so a socket task is started
-        comm.configure(state.hostName, state.portNumber);
-        comm.startSocketTask(
-            name,
-            true,
-            ConfigConstants::comm::PRIORITY,
-            ConfigConstants::comm::STACK_SIZE
-        );
-    }
-    """
-
-    phase Fpp.ToCpp.Phases.freeThreads """
-    comm.stopSocketTask();
-    (void) comm.joinSocketTask(nullptr);
-    """
-
-  }
+    at "../../Drv/TcpClient/TcpClient.hpp"
```

Comment:
Since including a header is a break in the typical pattern, add a comment to explain the difference.